website/events/services.py

```diff
@@ -294,6 +294,7 @@ def registration_fields(request, member=None, event=None, registration=None, nam
                 "type": field.type,
                 "label": field.name,
                 "description": field.description,
+                "short description": field.short,
```

Comment:
The API keys shouldn't contain spaces, as this can create some hassle. I propose you call this field "caption" everywhere, except maybe in the user messages